extensions/ARB/ARB_gl_spirv.txt

```diff
@@ -1149,12 +1149,16 @@ Add Appendix A.spv "The OpenGL SPIR-V Execution Environment" to the OpenGL 4.5
         be specified.
 
     Recursion:
-     - For all entry points, the static function-call graph rooted at that
-       entry point must not contain cycles.
+      - For all entry points, the static function-call graph rooted at that
+        entry point must not contain cycles.
 
     User declared input and output variables:
-     - Must not use OpTypeBool, either directly or as part of an aggregate.
+      - Must not use OpTypeBool, either directly or as part of an aggregate.
 
+    Atomic Access Rules:
+      - Atomic instructions must declare a scalar 32-bit integer type, or a
```

Comment:
I think we need more precise wording, regarding whether this is "built-in function", in which case it uses things, not declares them, or an "atomic type declaration".

Reply:
@johnkslang can you make a more specific suggestion?  This is in the SPIR-V execution environment, so talking about instructions seems correct to me.

Reply:
I see; I was missing the context of what section this is in.